app/controllers/tasks_controller.rb

```diff
@@ -1,45 +1,63 @@
 class TasksController < ApplicationController
-  load_and_authorize_resource :project
-  load_and_authorize_resource :task, through: :project
-
+  load_and_authorize_resource only: [:my_tasks, :update_task_logtime]
+  load_and_authorize_resource :project, except: [:my_tasks, :update_task_logtime]
+  load_and_authorize_resource :task, through: :project, except: [:my_tasks, :update_task_logtime]
+ 
   def create
     @task.set_status
     if @task.save
-      redirect_to project_tasks_url(@task.project), notice: "Returing from the create"
+      flash[:success] = 'Created Task Successfully!'
+      redirect_to project_tasks_url(@task.project)
     else
-      render new_project_task_path(@task.project), notice: "Cannnot enter data due to constraints"
+      flash[:danger] = 'Could not create Task!'
+      render new_project_task_path(@task.project)
     end
   end
 
   def update
     if !params[:assignable_employee_id].empty? && !params[:assignable_team_id].empty?
-      render :edit, notice: "Make sure you delete employee field or team field"
+      flash[:danger] = 'Make sure you delete employee field or team field'
+      render :edit
     else
-      params[:task][:status] = "new" if params[:task][:assignable_id].empty?
-      params[:task][:status] = "assigned" if !params[:task][:assignable_id].empty?
+      params[:task][:status] = 'new' if params[:task][:assignable_id].empty?
```

Comment:
hard coded strings

Reply:
This is already solved by abdullah in his branch, it'll be updated after merging